live-build/hooks/30-fix-timedatectl.chroot

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+
+set -e
+
+mv /usr/bin/timedatectl /usr/bin/timedatectl.real
+
+cat >/usr/bin/timedatectl<<EOF
```

Comment:
Hm, did you test this? A normal "heredoc" will substitute vars, so all the $1, $2, $@ will be empty in the output. You probably want `cat >/usr/bin/timedatectl<<'EOF'` instead.